src/packet_distribution.cpp

```diff
@@ -256,24 +256,30 @@ void AOApplication::server_packet_received(AOPacket *p_packet)
       // temporary. the CharsCheck packet sets this properly
       f_char.taken = false;
 
-      ++loaded_chars;
-
-      w_lobby->set_loading_text(tr("Loading chars:\n%1/%2")
-                                    .arg(QString::number(loaded_chars))
-                                    .arg(QString::number(char_list_size)));
+      if (!courtroom_loaded) {
+        ++loaded_chars;
+        w_lobby->set_loading_text(tr("Loading chars:\n%1/%2")
+                                      .arg(QString::number(loaded_chars))
+                                      .arg(QString::number(char_list_size)));
+      }
 
       w_courtroom->append_char(f_char);
```

Comment:
any significance keeping this in the middle? otherwise you could merge above and below if blocks together

Reply:
No significance, good idea. Tho due to being sick I'd ask you to do it for me if possible, my prs are branches on Master